internal/printer/printer_test.go

```diff
@@ -407,6 +407,16 @@ import Widget2 from '../components/Widget2.astro';`},
   </head><body></body></html>`,
 			},
 		},
+		{
+			name:   "text after title expression",
+			source: `<title>a {expr} b</title>`,
+			want: want{
+				imports:     "",
+				frontmatter: []string{},
+				styles:      []string{},
+				code:        `<html><head><title>a ${expr} b</title></head><body></body></html>`,
+			},
+		},
```

Comment:
This is passing now!

Reply:
nice! would be great if we could add another test for `<title>a {expr} b {expr} c</title>`